src/base/bittorrent/session.cpp

```diff
@@ -3044,13 +3049,18 @@ void Session::startUpTorrents()
         QByteArray data;
     } TorrentResumeData;
 
-    auto startupTorrent = [this, logger, resumeDataDir](const TorrentResumeData &params)
+    int addCount = 0;
+    auto startupTorrent = [this, logger, resumeDataDir, addCount](const TorrentResumeData &params) mutable
```

Comment:
* If you don't mind, please add `const` before `auto`.
* If I'm not mistaken, this should be `[this, &logger, &resumeDataDir, &addCount]` and remove the `mutable`
*